fuzzers/nyx_libxml2_parallel/README.md

```diff
@@ -0,0 +1,18 @@
+this example shows to use `libafl_nyx` to fuzz `libxml2`
```

Comment:
We usually have one `makefile.toml` for each fuzzer. You could just call the `.sh` file from it.